crates/dojo-world/src/lib.rs

```diff
@@ -112,6 +115,18 @@ impl EnvironmentConfig {
                 config.private_key = Some(pk);
             }
 
+            if let Some(path) =
+                env.get("keystore_path").and_then(|v| v.as_str().map(|s| s.to_string()))
```

Comment:
I think it would be good to fallback to env var also if it's not set in the manifest, following the same pattern as `private_key` and `account_address`.